db/db_local/backup.go

```diff
@@ -376,3 +380,55 @@ func getTableOfContentsFromBackup(ctx context.Context) ([]string, error) {
 
 	return lines, err
 }
+
+func PgDumpCmd(ctx context.Context, args ...string) *exec.Cmd {
+	cmd := exec.CommandContext(
+		ctx,
+		pgDumpBinaryExecutablePath(),
+		args...,
+	)
+	log.Println("[TRACE] pg_dump command:", cmd.String())
+	addLDPath(ctx, cmd)
+	return cmd
+}
+
+func PgRestoreCmd(ctx context.Context, args ...string) *exec.Cmd {
+	cmd := exec.CommandContext(
+		ctx,
+		pgRestoreBinaryExecutablePath(),
+		args...,
+	)
+	log.Println("[TRACE] pg_restore command:", cmd.String())
+	addLDPath(ctx, cmd)
+	return cmd
+}
+
+func addLDPath(ctx context.Context, cmd *exec.Cmd) {
```

Comment:
add comment - why are we doing this
Change name to UpdateDynamicLibPath

Reply:
done